src/Avalonia.Controls/Presenters/TextPresenter.cs

```diff
@@ -34,6 +34,9 @@ public class TextPresenter : Control
         public static readonly StyledProperty<IBrush?> CaretBrushProperty =
             AvaloniaProperty.Register<TextPresenter, IBrush?>(nameof(CaretBrush));
 
+        public static readonly StyledProperty<TimeSpan> CaretBlinkIntervalProperty =
+            AvaloniaProperty.Register<TextPresenter, TimeSpan>(nameof(CaretBlinkInterval), defaultValue: TimeSpan.FromMilliseconds(500));
+
```

Comment:
@Gillibald @maxkatz6 What is going on with Caret-related properties between TextPresenter and TextBox?

In a perfect world they should be factored out into a base class (which has been discussed elsewhere). Here though some properties are addowner from TextBox or vice-versa. TextPresenter seems lower level so actually TextBox should AddOwner this property. Overall, this really needs a thorough review and possibly noted for a v12 breaking change cleanup.

Then for this PR we need to decide if TextPresenter or TextBox should have the property and then AddOwner accordingly.

Reply:
@robloo @Gillibald @maxkatz6 If you all would like to do a major refactor of caret properties, that's likely out of the scope of this ticket. So you are welcome to close this PR and address it in a broader scope or merge this in the meantime- no hard feelings either way.

Reply:
No, as mentioned earliest would be v12 as it's a breaking change -- well after this PR. The property inheritance here (or lack thereof) doesn't make sense to me but need others feedback to confirm.

Reply:
Ah I see. I agree with that observation then. That area could use some work in the near future.

Reply:
TextPresenter is something I want to remove entirely. Syncing state via bindings is a total nightmare.